src/ArborX_LinearBVH.hpp

```diff
@@ -134,6 +134,10 @@ BoundingVolumeHierarchy<DeviceType>::BoundingVolumeHierarchy(
           Kokkos::ViewAllocateWithoutInitializing( "internal_and_leaf_nodes" ),
           _size > 0 ? 2 * _size - 1 : 0 )
 {
+    // This assumes that the _internal_and_leaf_nodes view allocation takes
+    // very little time.
```

Comment:
The name of the timer is "BVH construction". Thus, it is implied that this is how long it would take to construct it. If the view initialization takes too much time, it breaks this assumption.